docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3214,6 +3214,69 @@ In general, the same logic as is outlined for the census can be applied to all h
 
 For the two surveys currently outlined in the model, the default rates are 1.45% for ACS, and 29.05% for CPS.  
 
+**Duplicates:** 
+
+Duplication is most relevant for the census, which has a significant 
+amount of known duplicates. Often, the first PRL task is to match within 
+the same dataset in order to remove duplicates.  
+
+As a simplifying assumption in our initial model, we will be including 
+duplicates in the census only and limiting it to guardian-based duplication. 
+In later models, we might choose to include other forms of duplication with 
+more parameters.  
+
+**Guardian based duplication** 
+
+A known PRL challenge is children being reported multiple 
+times at different addresses. This can occur when family structures are 
+complex and children might spend time at multiple households. A related 
+challenge occurs with college students, who often are counted both at their 
+university and at their guardian’s home address.  
+
+To facilitate this type of error, we have simulants assigned to guardians 
+within the simulation. Sometimes, those guardians may move and live at 
+different addresses than their dependents. In this case, there is an 
+opportunity for duplication. Since this mechanism occurs within the 
+simulation, there is a natural maximum that we will impose in the 
+noise function.  
+
+Guardian based duplication is further divided here into two types: simulants 
+younger than 18 (<18), and those at college GQ less than 24 (<24).  
+
+For simulants younger than 18, an average percent who have a 
+guardian living at a different address will be calculated from the sim. 
+We will use 10% as a placeholder value. The user can then pick a rate of 
+duplication between 0 and 10%.  
+
+Based on this rate, the simulant will be duplicated, but with their address 
+changed to the guardian’s address instead of their “correct” address. 
+
+.. todo:: 
+
+    Find the maximum noise level from the simulation. If it is above ~25% or has a significant amount of variation over time or across ages, reconsider how to define the noise parameter.  
```

Comment:
Could we just have it be a conditional probability? i.e. the probability of being duplicated _given_ that you have a guardian in a different household. That would be naturally bounded between 0 and 100%.

Reply:
I did consider that, and it would certainly be easier. I felt that it would be too difficult for an end user to understand and lead to the obvious question of "well how many people are duplicated at 100%?". 

If finding an average proves too difficult, I think we will default to the conditional probability anyways. 

Do you think this is overly complex as is?

Reply:
Hm I could see somebody thinking about it either way, actually.

One idea: we could have the pseudopeople package dynamically calculate this maximum, and maybe make it only a warning if the user tries to set a value above the maximum.